src/firmware-update.html

```diff
@@ -0,0 +1,29 @@
+---
+layout: base
+description: Zigbee and Thread USB stick by the creators of Home Assistant
+---
+<div class="banner">
+  <span class="secondary">Web-Flasher</span>
+  <span>Firmware Updates for SkyConnect</span>
+</div>
+<div class="container">
+  <div>
+    <h1>Firmware update</h1>
+    <p>First things first: a firmware upgrade is not required for Home Assistant SkyConnect at this point.</p>
+    <p>We're working on making firmware updates available via Home Assistant. You will receive updates automatically in the future.
+      These updates are still under development.
+    </p>
+  </div>
+  <div><h3 class="text-left">SL Web Tools to flash firmware updates from the browser</h3></div>
+  <p>
+    If you want to have the latest update now: you can use the web-flasher below to install the latest firmware version for Home Assistant SkyConnect directly from your web-browser over USB. You can also use the web-flasher to flash the Zigbee firmware again if you've previously used the experimental multi-protocol support.    
```

Comment:
Nit. web[space]browser seems more common, but enough web already in that sentence IMHO.

```suggestion
    If you want to have the latest update now: you can use the web-flasher below to install the latest firmware version for Home Assistant SkyConnect directly from your browser over USB. You can also use the web-flasher to flash the Zigbee firmware again if you've previously used the experimental multi-protocol support.    
```